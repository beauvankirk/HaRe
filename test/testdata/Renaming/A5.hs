module Main where

import Renaming.B5
import Renaming.C5
import Renaming.D5

maim :: Tree Int ->Bool
maim t = isSame (sumSquares (fringe t))
               (sumSquares (Renaming.B5.myFringe t)+sumSquares (Renaming.C5.myFringe t))

main = putStrLn "hello"
