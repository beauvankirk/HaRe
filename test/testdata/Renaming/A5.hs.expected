module Main where

import Renaming.B5
import Renaming.C5
import Renaming.D5

maim :: Tree Int ->Bool
maim t = isSame (Renaming.D5.sum (fringe t))
               (Renaming.D5.sum (Renaming.B5.myFringe t)+Renaming.D5.sum (Renaming.C5.myFringe t))

main = putStrLn "hello"
